### PR TITLE
translation fix in thermal_sensors.widget.php

### PR DESCRIPTION
**Pull request by fabianfrz**

### CODE REVIEW

File: src/www/widgets/widgets/thermal_sensors.widget.php [hunk 1]
```diff
@@ -267,8 +267,7 @@ function getBoolValueFromConfig(&$configArray, $valueKey, $defaultValue)
 		</tr>
 		<tr>
 		<td colspan="3">
-      <span>* <?= gettext('You can configure a proper Thermal Sensor / Module under <br />
-    &nbsp;&nbsp;&nbsp;<a href="system_advanced_misc.php">System &gt; Advanced &gt; Miscellaneous : Thermal Sensors section</a>.') ?></span>
+      <span>* <?= sprintf(html_safe(gettext('You can configure a proper Thermal Sensor / Module under %s%sSystem > Advanced > Miscellaneous : Thermal Sensors section%s.')),'<br />&nbsp;&nbsp;&nbsp;','<a href="system_advanced_misc.php">','</a>') ?></span>
```
Comment (fichtner) [inline src/www/widgets/widgets/thermal_sensors.widget.php]: No more breaking this into two lines + drop implication of "System: Advanced: Misc" as it hard to translate consistently and may be subject to change.

File: src/www/services_dhcp.php [hunk 2]
```diff
@@ -985,7 +985,7 @@ function show_netboot_config() {
 											</div>
 											<div id="showldap" style="display:none">
 												<input name="ldap" type="text" class="form-control unknown" id="ldap" size="80" value="<?=htmlspecialchars($pconfig['ldap']);?>" /><br />
-												<?=gettext("Leave blank to disable.  Enter a full URI for the LDAP server in the form ldap://ldap.example.com/dc=example,dc=com");?>
+												<?=gettext("Leave blank to disable. Enter a full URI for the LDAP server in the form ldap://ldap.example.com/dc=example,dc=com");?>
```
Comment (fichtner) [inline src/www/services_dhcp.php]: Can we have the sample URI as %s , makes it easier not to screw up the translation by accident.

File: src/www/services_dhcp_edit.php [hunk 3]
```diff
@@ -519,7 +519,7 @@ function show_tftp_config() {
 									</div>
 									<div id="showtftp" style="display:none">
 										<input name="tftp" type="text" class="formfld unknown" id="tftp" size="50" value="<?=htmlspecialchars($pconfig['tftp']);?>" /><br />
-										<?=gettext("Leave blank to disable.  Enter a full hostname or IP for the TFTP server.");?>
+										<?=gettext("Leave blank to disable. Enter a full hostname or IP for the TFTP server.");?>
```
Comment (fichtner) [inline src/www/services_dhcp_edit.php]: Small note on these ".  " type things: it's not a mistake, this is old typewriter end of sentence style still employed in technical BSD manuals. But I agree that we should migrate them bit by bit to what you've done here and elsewhere. thanks :)

**Comment by fichtner** [comment]
After the URI change I will merge. :)

**Comment by fabianfrz** [comment]
done

**Comment by fichtner** [comment]
Neat, thanks Fabian :)

**Pull request merged**